ml5js/ml5-library
# Specify webpack-dev-server port

I noticed some issues running `npm run develop` regarding the port that the examples attempt to load their ml5 version from in development. This PR specifies port 8080 for our webpack-dev-server that builds ml5.js to that we don't have those issues.

Test Plan
- `npm run develop`
- Check a hosted example (i.e. not a p5js web editor link)
- Make sure that it actually loads without any issues loading the locally built ml5.js build.
